tests/manage/storageclass/test_delete_storageclusterinitialization.py

```diff
@@ -0,0 +1,70 @@
+import logging
+
+from ocs_ci.ocs import constants, defaults
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.framework.testlib import (
+    ManageTest, tier4, polarion_id, ignore_leftovers
+)
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier4
```

Comment:
@jilju I guess you didn't add tier4a/b/c since you don't want this test to be executed right?